test.js

```diff
@@ -0,0 +1,45 @@
+import {serial as test} from 'ava';
```

Comment:
Have to run them serially at the moment. Probably going to change that as it's a bug in the mock context. Going to create an issue over there to discuss this first. So feel free to keep it open untill we resolve this.